pubspec.yaml

```diff
@@ -14,7 +14,7 @@ author: 'Dart Team <misc@dartlang.org>'
 homepage: https://github.com/dart-lang/stack_trace
```

Comment:
☝️ feel free to drop author while you're here

Reply:
Handled in https://github.com/dart-lang/stack_trace/pull/58

Reply:
👍